statics/js/components/topology.js

```diff
@@ -37,6 +33,10 @@ var TopologyComponent = {
               </i>\
             </span>\
           </button>\
+          <button id="expand" type="button" class="btn btn-primary" \
+                  title="Expand" @click="toggleCollapseByLevel(false)">++</button>\
```

Comment:
I think we can reuse the same icon as it means expand and the context is different or at least we can find something else. I think using ++/-- break the look and feel and lead to think adding/deleting.

Reply:
if we use the same icon it will confuse the user. I will do one thing, use the icon for step by step expand and collapse. use a word button for complete tree expand as of now. later we can change with the appropriate icon. what do you think?

Reply:
what do you mean by "tree expand" I didn't get this point

Reply:
i am using fa-expand and fa-collapse icons here(https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L32) so I can't use the same icons instead of "++" and "--". my question was shall i use fa icons for ++ and -- and use text in https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L32

Reply:
ok got your point. I don't find this really confusing but I would propose to keep the tree expand as it is right now and to use new icons for '++'/'--'. What do you think about this
http://jsfiddle.net/3k3u5qs4/
it combines two icons